test/test_torch.py

```diff
@@ -6996,11 +6984,6 @@ def _test_logical(self, device, op, a_, b_, expected_res_):
 
             # in-place
             b = torch.tensor(b_, dtype=dtype, device=device)
-            # Skip bfloat16 on CUDA. Remove this after bfloat16 is supported on CUDA.
-            if device.startswith('cuda') and dtype == torch.bfloat16:
```

Comment:
This will probably have to stay, since you're not adding support for these ops on CUDA.